### DISCUSSION TITLE
Problems generating corefiles with WSL2

### DISCUSSION THREAD

**Discussion by paul-haskell** [microsoft/WSL#11992] [Q&A]
Hi all -- I am trying to generate a corefile under WSL2.
1) I disabled apport (sudo service apport stop)
2) I set the kernel.core_pattern appropriately (sudo sysctl kernel.core_pattern=core.%e.%p)
3) I set the corefilesize limit to 'unlimited'
4) I verified the current directory is writable by all
5) I ran a few simple programs that should throw a core (abort(), integer divide-by-0)

...but I never get a core.  I do reliably get a corefile on a native Ubuntu machine.

Does anyone have any ideas to try?  Thanks!

(I am running WSL version 2.2.4.0 with default Ubuntu i.e. 22.04.3 LTS.  I am running Windows version 10.0.22631.4037 .)

**Comment by OneBlue** [comment]
@paul-haskell: Coredumps are now automatically generated in WSL2. They will be located in `C:\Users\<user>\AppData\Local\Temp\wsl-crashes`
